activestorage/lib/active_storage/attached/changes/create_one.rb

```diff
@@ -82,6 +82,18 @@ def find_or_build_blob
           )
         when String
           ActiveStorage::Blob.find_signed!(attachable, record: record)
+        when File, Pathname
+          io = attachable.is_a?(File) ? attachable : attachable.open
+          filename = File.basename(attachable)
+          content_type = Marcel::MimeType.for(io, name: filename)
+
+          ActiveStorage::Blob.create_and_upload!(
```

Comment:
Maybe use `build_after_unfurling` instead of `create_and_upload!` like the other branches. Plus, the method's name is `find_or_build_blob`

Reply:
Good call, I don't know why I was using `create_and_upload!`, `build_after_unfurling` works well